NOTE.md

```diff
@@ -0,0 +1,20 @@
+### Introduce `tflite-infer` tool
+
+It's used to infer tflite model and get the result data.
+`tflite-infer` code itself is included in [#9122](https://github.com/Samsung/ONE/pull/9122) too. This pr includes tests, manpage, cmake codes.
+
+### test design description
+
+Category | Test # | Description
+:--: | :--: | :--
+positive test | P-01 | -h printing test
+positive test |	P-02	|-l --input-spec npy:\<filename> execution & output npy dump value test
+positive test |	P-03	|-l --input-spec h5:\<filename> execution & output h5 dump value test
+positive test |	P-04	|-l --input-spec any input npy&h5 shape/dtype test
+positive test|	P-05	|-l --input-spec non-zero input npy&h5 value, shape/dtype test
+positive test|	P-06	|-l --input-spec positive input npy&h5 value, shape/dtype test
```

Comment:
Current on-going local draft's approach:

Use simple add tflite model (which is already quantized to int8) ,pre-generated random input file(both npy and h5) and golden output data file. Those will be prepared while `prepare_test_materials` stage. 
Save the `tflite-infer`'s output data and compare the md5sum hash with golden.

```
Simple add model:
LHS ---\
        +--- ADD --- OUT
RHS ---/
```